prerequisites/install-functions/build_opencoarrays.sh

```diff
@@ -22,9 +22,16 @@ build_opencoarrays()
   FC="${MPIFC_show[0]}"
   # Set CC to the MPI implementation's gcc command...
   CC="${MPICC_show[0]}"
+  # try to find mpiexec
+  MPIEXEC_CANDIDATES=($(find "${MPICC%/*}" -name 'mpiexec' -o -name 'mpirun'))
```

Comment:
Assumes MPICC set correctly, and searches for `mpiexec` or similar in same directory as compiler wrapper.